docs/pages/server-access/guides/ec2-discovery.mdx

```diff
@@ -21,7 +21,42 @@ install Teleport manually.)
 
 (!docs/pages/includes/tctl.mdx!)
 
-## Step 1/5. Define an IAM policy
+## Step 1/6. Create an EC2 invite token
+
+When discovering EC2 instances, Teleport makes use of IAM invite tokens for
```

Comment:
Amended in https://github.com/gravitational/teleport/pull/18308/commits/1762ae809414776cd68cf3db508f27bb96b8ab89

Reply:
Until this point, there's nothing to indicate how EC2 Discovery works and why we need an invite token. I think a sentence about how these tokens work within a Teleport cluster would go a long way toward explaining, say, why we would want to set a longer TTL on the token. For example,  does the SSH Service poll the EC2 APIs in the configured regions until the token TTL elapses? 

We also don't mention SSM until toward the end of the guide, so it's not really clear architecturally speaking what EC2 Auto-Discovery looks like unless the reader figures it out on their own as they go through the guide.

Also, per the [style guide](https://goteleport.com/docs/contributing/documentation/style-guide/#getting-started-articles), we're trying to avoid external links in tutorials. If we're trying to get the user through the tutorial as quickly as possible so they can see value from Teleport, telling them to read another docs page without a clear goal for doing so adds time to complete the guide and risks distracting the user because it requires shifting context.

Reply:
That sounds good to me!